packages/kbn-alerts-ui-shared/src/rule_type_modal/components/rule_type_modal.tsx

```diff
@@ -121,7 +121,6 @@ export const RuleTypeModal: React.FC<RuleTypeModalProps & RuleTypeModalState> =
           <EuiFlexItem
             style={{
               overflow: 'hidden',
-              marginTop: -euiTheme.size.base /* Offset extra padding for card hover drop shadow */,
```

Comment:
I checked the modal in observability, and if I remove "-", it adds more space than the current implementation; please let me know if you see an issue with this change. (euiTheme.size.base value is 16px)

Reply:
> it seems like it's being transpiled to marginTop: NaN

Yes, that's exactly the issue that I am solving. I meant if we apply it it will change the spacing from the current version but I guess the current one is wrong, so I fixed it in [13fb558](https://github.com/elastic/kibana/pull/194766/commits/13fb55857f74e7ee53a32890eab7b727e09701eb)